### ISSUE TITLE
Installation and setup documentation is missing

### ISSUE THREAD

**Issue by blami** [neoclide/coc.nvim#923]
**Is your feature request related to a problem? Please describe.**
I tried to get coc working with my nvim environment but I failed. There's no good document to go from zero to working setup. After installation (using limited cryptic steps in README.md) I'm left with nvim showing completion window on every pres of spacebar completing random tokens from same file only.

**Describe the solution you'd like**
I'd like to read through document that describes how to setup coc that behaves similar to e.g. visual studio or emacs autocompletion (popup only when I type language accessor or pres key combination) with relevant only tokens.

**Describe alternatives you've considered**
-

**Additional context**
I have only basic understanding of vim/neovim, can't script in vimscript nor have any tribal knowledge. I only use neovim as a tool to code.

**Comment by chemzqm** [comment]
Have you read https://github.com/neoclide/coc.nvim/wiki/Install-coc.nvim?
Please explain what blocks you.

**Comment by blami** [comment]
@chemzqm Yes I did. After following all steps in that document I was left exactly in described state where e.g. opening go source resulted in autocomplete giving me random words from that file only. 

There's no slight mention in this document how should I configure coc to complete various languages. On reddit I found I need to create configuration file coc-settings.json and install additional software, that is fain except it is not mentioned in documentation referenced by you at all. I later found reference to this configuration file in Wiki, however there's no clear explanation of what each configuration directive does. 

I'm not saying documentation is missing, I'm just saying there's unfortunately no newb-friendly documentation.

**Comment by chemzqm** [comment]
Here's example configuration for go: https://github.com/neoclide/coc.nvim/wiki/Language-servers#go

An extension would make it a lot easier.

**Comment by blami** [comment]
Point of this issue was more about friendlyness than incompletness of the documentation. For example this page doesn't even say where someone who never heard about coc-settings.json should put those lines.

Anyway thanks for help!

**Comment by eikenb** [comment]
I think the wiki is a pretty good start and like all wiki pages will get fleshed out over time. I'm commenting on this issue because the wiki page isn't obvious when first looking ( to myself at least) due to the Installation section in the Readme/Table-Of-Contents having content unlike all the other entries. My suggestion would be to remove some/all of the partial installation instructions in the Readme and just direct people to the wiki like the other topics in the table-of-contents.
